tests/parsers/perkin_elmer_envision/testdata/PE_Envision_fluorescence_example01.json

```diff
@@ -5485,1019 +5485,1019 @@
             "software name": "EnVision Workstation",
             "software version": "1.14.3049.1193",
             "ASM converter name": "allotropy",
-            "ASM converter version": "0.1.10"
+            "ASM converter version": "0.1.12"
         },
         "calculated data aggregate document": {
             "calculated data document": [
                 {
-                    "calculated data name": "(X / Y) * Z where X = AC HTRF Laser [Eu](1) channel 1 window 1  Y = AC HTRF Laser [Eu](1) channel 2 window 1  Z = 10000",
+                    "calculated data name": "Calc 1: General",
```

Comment:
it seems like you could find/replace "calculated data name" value in the test files and prevent the "measurement identifier" changes, making the test changes clearer, can you do that please?